_posts/2019-04-17-sod.md

```diff
@@ -0,0 +1,58 @@
+
+**Related material:**
+
+1. https://ci.apache.org/projects/flink/flink-docs-release-1.8/
+2. https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev
+3. https://ci.apache.org/projects/flink/flink-docs-release-1.8/ops/
+4. https://ci.apache.org/projects/flink/flink-docs-release-1.8/concepts/programming-model.html#time
+5. https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/event_time.html
+
+### Project 2: Improve Documentation of Flink Deployments & Operations?
+
+**Description:** Stream processing is the processing of data in motion―in other words, computing on data directly as it is produced or received. Apache Flink has pioneered the field of distributed, stateful stream processing for the last few years. As a stateful distributed system in general and a continuously running, low-latency system in particular, Apache Flink deployments are non-trivial to setup and manage.
+Unfortunately, the operations \[1\] and monitoring documentation \[2\] are arguably the weakest spots of the Apache Flink documentation. While it is comprehensive and often goes into a lot of detail, it lacks an overall structure and does not address common overarching concerns of operations teams in an efficient way.
+
+In this project, we would like to restructure this part of the documentation and extend it if possible. Ideas for extension include: discussion of session and per-job clusters, better documentation for containerized deployments (incl. K8s), capacity planning &  integration into CI/CD pipelines.
+
+**Related material:**
+
+1. https://ci.apache.org/projects/flink/flink-docs-release-1.8/ops/
+2. https://ci.apache.org/projects/flink/flink-docs-release-1.8/monitoring
+
+### Project 3: Improve Documentation for Relational APIs (Table API & SQL)?
+
+**Description:** Apache Flink features APIs at different levels of abstraction which enables its users to trade conciseness for expressiveness. Flink’s relational APIs, SQL and the Table API, are “younger” than the DataStream and DataSet APIs, more high-level and focus on data analytics use cases. A core principle of Flink’s SQL and Table API is that they can be used to process static (batch) and continuous (streaming) data and that a program or query produces the same result in both cases.
+The documentation of Flink’s relational APIs has organically grown and can be improved in a few areas. There are several on-going development efforts that aim to extend the scope of the Table API and SQL.
+
+The existing documentation could be reorganized to prepare for covering the new features. Moreover, it could be improved by adding a concepts section that describes the use cases and internals of the APIs in more detail. Moreover, the documentation of built-in functions could be improved by adding more concrete examples.
```

Comment:
Should we list some new features on roadmap? For example, DDL, Hive integration, Python Table API, Interactive Programming. IMO, listing some interesting features may can help to attract writers.

Reply:
What we are doing now is document improvement, so, from points of my view, it is reasonable to describe it based on existing content. :)  But I think you mentioned a good point.

Reply:
I added some examples.